ibtk/src/utilities/ibtk_utilities.cpp

```diff
@@ -48,4 +48,19 @@ get_min_patch_dx(const PatchLevel<NDIM>& patch_level)
 
     return result;
 } // get_min_patch_dx
+
+bool
+rel_equal_eps(const double a, const double b, const double eps)
+{
+    if (a == b) return true;
```

Comment:
```suggestion
    return (a == b) || ((std::abs(a - b) / std::max(std::abs(a), std::abs(b))) < eps);
```
This will "short circuit" if `a == b` and not evaluate the RHS of the `||`.

Reply:
BTW, I am not sure if the extra equality check is worth doing or not (in either of these functions) --- might be worthwhile to do some timing checks.

Reply:
The equality check here is to prevent dividing by zero. I don't think it's worth doing in the other function.

If speed is truly a concern, we could implement a check to see if `a` is within some number of representable numbers away from `b`. This would remove the division but still be similar to a relative difference check.